app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1248,4 +1248,13 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         featureFlag = FeatureFlags.isPocketRecommendationsFeatureEnabled(appContext),
         default = { appContext.components.analytics.features.homeScreen.isPocketRecommendationsActive() },
     )
+
+    /**
+     * Indicates if the Contile functionality should be visible.
+     */
+    var showContileFeature by lazyFeatureFlagPreference(
+        appContext.getPreferenceKey(R.string.pref_key_enable_contile),
+        default = { false },
```

Comment:
```suggestion
        default = false,
```

Looks like this can be `false`

Reply:
One would think anyway
![image](https://user-images.githubusercontent.com/87384386/150029955-eff7ca38-be61-4fb8-a224-dbfdfb6829c6.png)